.gitignore

```diff
@@ -59,6 +59,8 @@ Vagrantfile
 /.tmp/
 /vendor/bundle
 
+saml_*.txt
```

Comment:
Can we make it this?   
saml_*.???

Reply:
Or even more precise: saml_??????????????.???

Reply:
I applied it to our current use case. If we come across other variations, we can certainly update it.

Reply:
Are you saying what I added won't catch all variations?

Reply:
Correct but the specs should be cleaning up afterwards.  I could also mock the file writes.

Reply:
That would be great. I opened a related JIRA issue (461) about silencing all the noise that several tests emit.